### ISSUE TITLE
rrdb是预训练的模型，请问它的ckpt在哪下载

### ISSUE THREAD

**Issue by Lincoln20030413** [LeiaLi/SRDiff#6]
你好，感谢你们的工作，我有一个问题，你在论文中提到RRDB based LR encoder D is pretrained by L1 loss function，请问它的预训练模型的ckpt文件在哪？

**Comment by poxiao0** [comment]
+1

**Comment by duanpanjun** [comment]
> 你好，感谢你们的工作，我有一个问题，你在论文中提到RRDB based LR encoder D is pretrained by L1 loss function，请问它的预训练模型的ckpt文件在哪？

请问你找到了吗？我也没找到在哪
